### PR TITLE
test: uniswapx e2e activity

### PR DESCRIPTION
**Pull request by cartcrom**
## Description

Tests activity history and balance fetching in mini portfolio following uniswapx order submission / completion

### CODE REVIEW

File: src/components/AccountDrawer/PrefetchBalancesWrapper.tsx [hunk 1]
```diff
@@ -3,36 +3,19 @@ import { usePortfolioBalancesLazyQuery, usePortfolioBalancesQuery } from 'graphq
...
+  const prevPending = usePrevious(totalPending)
 
-  return useMemo(() => {
-    if (!previousPendingTxs || !account) return false
-    return Object.values(currentChainTxs).some(
-      (tx) => tx.from === account && !isTxPending(tx) && wasPending(previousPendingTxs, tx),
-      [currentChainTxs, previousPendingTxs]
-    )
-  }, [account, currentChainTxs, previousPendingTxs])
+  return !!prevPending && totalPending < prevPending
```
Comment (cartcrom) [inline src/components/AccountDrawer/PrefetchBalancesWrapper.tsx:18]: I realized this hook was very overcomplicated and could be replaced w/ just counting difference in pending activity count between renders

Comment (tinaszheng) [reply src/components/AccountDrawer/PrefetchBalancesWrapper.tsx:18]: nice

File: src/state/transactions/hooks.tsx [hunk 2]
```diff
@@ -150,5 +150,10 @@ export function isPendingTx(tx: TransactionDetails): boolean {
 
 export function usePendingTransactions(): TransactionDetails[] {
   const allTransactions = useAllTransactions()
-  return useMemo(() => Object.values(allTransactions).filter(isPendingTx), [allTransactions])
+  const { account } = useWeb3React()
+
+  return useMemo(
+    () => Object.values(allTransactions).filter((tx) => tx.from === account && isPendingTx(tx)),
```
Comment (tinaszheng) [inline src/state/transactions/hooks.tsx:156]: nice

**Review by tinaszheng — APPROVED** [review]
🔥

**Review by just-toby — APPROVED** [review]

**Review by tinaszheng — APPROVED** [review]

**Review by just-toby — APPROVED** [review]

**Pull request merged**